odysseus/convertFtbQuests.ts

```diff
@@ -70,7 +74,7 @@ type QuestTask = QuestObject & ({
 
     item: Item;
 
-    count?: number;
+    count?: string;
```

Comment:
What's the original type here in the SNBT?

Reply:
I would love to have the answer to that question:

![image](https://github.com/terrarium-earth/odysseus/assets/55819817/b4bb3f92-d948-4357-814c-227c646acfb4)

![image](https://github.com/terrarium-earth/odysseus/assets/55819817/059b06c3-5444-4abc-bba9-3b1cc8146f92)

![image](https://github.com/terrarium-earth/odysseus/assets/55819817/edac0273-a509-4547-a058-e048931ef74b)

in practice, I know I had to manually replace stringified counts, amounts, and lvls with a pure number in json, so i'm playing defence.

parseInt(count.tostring()) would be fine too.

Reply:
should use the `Long` utility type

Reply:
done in 0120036